algorithms/linfa-pls/src/pls_generic.rs

```diff
@@ -148,6 +148,10 @@ impl<F: Float, D: Data<Elem = F>> PredictInplace<ArrayBase<D, Ix2>, Array2<F>> f
         x /= &self.x_std;
         *y = x.dot(&self.coefficients) + &self.y_mean;
     }
+
+    fn num_targets(&self) -> usize {
+        todo!()
```

Comment:
What goes here?

Reply:
you can look at what is assigned to your targets. `y` here has (#obserations, #targets) and `self.y_mean` is added, so you can use `self.y_mean.ncols()`